content/11.interpretation.md

```diff
@@ -1 +1,7 @@
-## Tip 9: Interpret predictions in the correct manner {#interpretation}
+## Tip 9: Don't over-interpret predictions {#interpretation}
+
+Deep learning models can make predictions with high accuracy, but we need to take care to correctly interpret these predictions. We know that the basic tenets of machine learning also apply to deep learning (Tip #1), but because deep models can be difficult to interpret intuitively, there is a temptation to anthropomorphize deep models. We must resist this temptation.
```

Comment:
We can link to the other rules with relative links.  For example, `(see [Tip 1](#concepts))`, will link to the Tip 1 subsection.  The top of each tip's markdown file has its link syntax, e.g. `{#concepts}`.